internal/stacks/stack.go

```diff
@@ -23,6 +24,12 @@ import (
 var homeDir, _ = os.UserHomeDir()
 var StacksDir = path.Join(homeDir, ".firefly", "stacks")
 
+//go:embed ganache/Dockerfile
```

Comment:
Nice to know this is there - can think of a few places I should have used this 👍